src/TwigComponent/doc/index.rst

```diff
@@ -1719,6 +1719,36 @@ Pass the name of some component as an argument to print its details:
     |                                                   | int $min = 10                     |
     +---------------------------------------------------+-----------------------------------+
 
+Short Syntax
```

Comment:
This is **not** a new syntax.

We really should be precise on this :) Having _two_ syntaxes is already something that did create its fair share of frustrations, documetation problems, tests, etc etc ..  let's not frighten or confuse anyone with no reason here :)